### ISSUE TITLE
Not converge problem

### ISSUE THREAD

**Issue by TianrongChen** [anassinator/pddp#1]
Hi, thanks for your implement of PDDP. But when I run your code, none of them converge. Maybe because I do not give it enough time? By the way, the jupyter notebook you provided has bugs XD
Best

**Comment by anassinator** [comment]
Yeah this was a work in progress and was never completed. It definitely doesn't work yet :P I'd gladly accept a PR for bug fixes.
